upup/pkg/fi/cloudup/awstasks/launchtemplate_target_api.go

```diff
@@ -51,6 +51,13 @@ func (t *LaunchTemplate) RenderAWS(c *awsup.AWSAPITarget, a, e, changes *LaunchT
 		},
 	}
 
+	if t.HTTPPutResponseHopLimit != nil || t.HTTPTokens != nil {
+		data.MetadataOptions = &ec2.LaunchTemplateInstanceMetadataOptionsRequest{
+			HttpPutResponseHopLimit: t.HTTPPutResponseHopLimit,
+			HttpTokens:              t.HTTPTokens,
+		}
+	}
+
```

Comment:
You'll need to read the value back from AWS in the `Find()` method below in order to prevent extraneous changes reported during subsequent `kops update cluster` dryruns. It should just involve setting `actual.HTTPTokens` and `actual.HTTPPutResponseHopLimit` based on the contents of `lt`.

Reply:
ah got it. Will do this.

Will get back to you if I need some help with the cloudmocks :)